packages/product-react/src/index.ts

```diff
@@ -0,0 +1,3 @@
+import '@wework/ray-core/scss/ray-core.scss';
```

Comment:
this should be removed. 

the responsibility of loading the styles is on the consumer